recipes/ctng-compiler-activation/activate-binutils.sh

```diff
@@ -0,0 +1,116 @@
+#!/bin/bash
```

Comment:
Should be `/usr/bin/env bash`

Reply:
Activate scripts are sourced and it doesn't matter. This script supports both `bash` and `tsh`